src/main/scala/util/RunUtils.scala

```diff
@@ -143,16 +163,20 @@ object IRLoading {
               ${line.replace('\n', ' ')}
               ${" " * token.getStartIndex}^ here!
               """.stripIndent
-            case _ => ""
+            case o => o.toString
           }
           Logger.error(s"""Semantics parse error:\n  line: $line\n$extra""")
           throw e
       }
     }
 
-    val parserMap = semantics.map(_.map((k: String, v: Array[Array[String]]) => (k, v.map(_.map(parse_insn)))))
+    val parserMap : Map[String, List[Either[(String,String), List[StmtContext]]]] =
```

Comment:
(String, String) should probably be a case class instead of a tuple to make the semantics of it clearer